<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPTDemo                   作者:WildPointer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>答复信息</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="reply_Text"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>操作框</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="question_Text"/>
     </item>
     <item>
      <widget class="QPushButton" name="enquire_Btn">
       <property name="text">
        <string>开始询问</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>设置AI</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="key_Text"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="secret_Text"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>设置Key</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>设置Secret</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="set_API">
       <property name="text">
        <string>设置鉴权元素</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
